README.md

```diff
@@ -175,38 +183,48 @@ $ java -jar cromwell.jar run my_wf.wdl - - my_wf.metadata.json
 ... play-by-play output ...
 $ cat my_wf.metadata.json
 {
+  "workflowName": "w",
   "calls": {
-    "example.my_task": [{
+    "w.x": [{
       "executionStatus": "Done",
```

Comment:
Although it's a problem because sometimes the instantiated command is actually huge due to hundreds of input parameters, which may cause the PutMetadata to fail at the DB level (happened to me when running simple sample).